lib/appium_lib/device/device.rb

```diff
@@ -15,6 +15,7 @@ module Device
         device_locked?:       'session/:session_id/appium/device/is_locked'
       },
       get:  {
+        get_device_time:        'session/:sessionId/appium/device/system_time',
```

Comment:
`:sessionId` is wrong. routes.js is JavaScript, not Ruby. Device methods in Ruby are specified like this:

> session/:session_id/appium/device/